_build/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/relrod/ansible-signatory
```

Comment:
just realize this... we should change it to `https://github.com/ansible/ansible-sign`